src/main/java/com/EstimatedDeliveryDate/Entity/Holiday.java

```diff
@@ -0,0 +1,41 @@
+package com.EstimatedDeliveryDate.Entity;
+
+import java.io.Serializable;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBDocument;
+
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/*Document object for the Holiday List*/
+
+@Setter
+@NoArgsConstructor
+@DynamoDBDocument
+public class Holiday  implements Serializable {
+	private static final long serialVersionUID = 1L;
+	private String holidayDate;
```

Comment:
Similar comment as CPT: are we planning to change this to date object in business logic or here itself?

Reply:
Yes